services/openuserjs/openuserjs-downloads.service.js

```diff
@@ -0,0 +1,27 @@
+import { renderDownloadsBadge } from '../downloads.js'
+import BaseOpenUserJSService from './openuserjs-base.js'
+
+export default class OpenUserJSDownloads extends BaseOpenUserJSService {
+  static category = 'downloads'
+  static route = { base: 'openuserjs', pattern: 'dt/:username/:scriptname' }
+
+  static examples = [
+    {
+      title: 'OpenUserJS',
+      namedParams: {
+        username: 'NatoBoram',
+        scriptname: 'YouTube_Comment_Blacklist',
+      },
+      staticPreview: renderDownloadsBadge({ downloads: 47 }),
+    },
+  ]
+
+  static defaultBadgeData = { label: 'downloads' }
+
+  async handle({ username, scriptname }) {
+    const data = await this.fetch({ username, scriptname })
+    return renderDownloadsBadge({
+      downloads: data.OpenUserJS.installs[0].value,
```

Comment:
So for symmetry sake since you did the `at` prototype elsewhere for the UserScript block *(love when they sneak these methods in and I miss them btw)* do you think you ought to do it on _all occurrences_ in this PR that are applicable? It's not a huge deal breaker but might be nice to have.

Reply:
As there's only 1 item as of now, they seem to be equivalent, although I can do that for consistency :+1: